### PR TITLE
Remove LyShineExample Gem's dependency on LmbrCentral and address feedback from previous PR with similar fix for ScriptCanvas

### PR DESCRIPTION
**Pull request by amzn-sj**
Signed-off-by: amzn-sj <srikkant@amazon.com>

### CODE REVIEW

**Review by amznestebanpapp — APPROVED** [review]
Thanks!

**Review by AMZN-Phil — APPROVED** [review]

**Review by lsemp3d — APPROVED** [review]

File: Gems/LmbrCentral/Code/lmbrcentral_headers_files.cmake [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    include/LmbrCentral/Shape/DiskShapeComponentBus.h
+    include/LmbrCentral/Shape/CompoundShapeComponentBus.h
+    include/LmbrCentral/Shape/QuadShapeComponentBus.h
+    include/LmbrCentral/Shape/SplineComponentBus.h
+    include/LmbrCentral/Shape/PolygonPrismShapeComponentBus.h
+    include/LmbrCentral/Shape/TubeShapeComponentBus.h
+    include/LmbrCentral/Shape/SplineAttribute.h
+    include/LmbrCentral/Shape/SplineAttribute.inl
+    include/LmbrCentral/Terrain/TerrainSystemRequestBus.h
+)
```
Comment (AMZN-alexpete) [inline Gems/LmbrCentral/Code/lmbrcentral_headers_files.cmake:63]: can we add a newline at the end of this file for linux?

**Review by AMZN-alexpete — APPROVED** [review]
Just missing a new line at the end of the Gems/LmbrCentral/Code/lmbrcentral_headers_files.cmake  file

File: Gems/LmbrCentral/Code/lmbrcentral_headers_files.cmake (hunk 1 above)
Comment (amzn-sj) [reply Gems/LmbrCentral/Code/lmbrcentral_headers_files.cmake]: Done

**Pull request merged**